src/modules/awake/Awake/Core/Models/SystemPowerCapabilities.cs

```diff
@@ -53,8 +53,6 @@ public struct SystemPowerCapabilities
         public byte HiberFileType;
         [MarshalAs(UnmanagedType.U1)]
         public bool AoAcConnectivitySupported;
-        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 6)]
-        private readonly byte[] spare3;
```

Comment:
Why is this deleted? Shouldn't this still be part of the struct, according to https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-system_power_capabilities and https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/System/Power/struct.SYSTEM_POWER_CAPABILITIES.html?

Reply:
Another q, why is it private when others are public?

Reply:
@crutkas @jaimecbernardo notice in the docs it's a reserved and seemingly unused property. No reason for it to be in the struct at this time.

Reply:
@dend , When you call a native method with a structure, it's important that they match what's expected by the native dll. Native methods don't see the names in your structure, they just expect the size and layout to match what the structure they are using internally. So when you delete this field, there's possible memory overflow or the call saves fields in the wrong place in the structure. Hope this makes sense. Can we please revert the change?
@crutkas it's private because the documentation marks it as Reserved (internal use by the API or other APIs).

Reply:
For what it's worth, @jaimecbernardo - then we need to _expand_ this structure because it's missing a few other fields. Happy to do that, but just as a heads-up that this struct was incomplete as-is.

Reply:
From what I see it's only used for logging for debugging?
![image](https://github.com/user-attachments/assets/ebe61381-6edb-4987-a0f4-620222cbe2e6)
If that's the case probably means those logs might have been receiving garbage 😅

Reply:
@jaimecbernardo you actually just now helped me identify a bug - the JSON was never produced and everything there was captured as `{}`. Thanks for the heads-up! This is now fixed in 223b930180910bde27ab20b2dd9c3c7f476ce321.